source/installguide/hypervisor/kvm.rst

```diff
@@ -317,19 +317,19 @@ cloudstack-agent and should already be installed.
 
       env libvirtd_opts="-d -l"
 
-   On Ubuntu 16.04: just modify ``/etc/init/libvirt-bin.conf``
+   On Ubuntu 16.04: just modify ``/etc/default/libvirt-bin``
 
-   Add "-l" to the following line
+   Uncomment and change the following line
 
    .. parsed-literal::
 
-      env libvirtd_opts="-d"
+      #env libvirtd_opts=""
```

Comment:
@rhtyd better to provide possible options / recommendations. your opinion?

Reply:
@sureshanaparti what do you mean?

Reply:
@bwsw i mean libvirtd_opts

Reply:
if any, update the comment. else fine.

Reply:
@sureshanaparti I don't get. Just read several lines. It states
1. uncomment line
2. add "-l"
done.
Could you please explain what you expect?

Reply:
@sureshanaparti you may think, I'm kidding you... But I'm not. No offense, but I really, don't get what do you mean. Try use code more verbal and detailed explanation. You may also want open new PR with a better fix for that.
@rhtyd @rafaelweingartner if you understand what @sureshanaparti is talking about, please help... because I don't.

Reply:
@bwsw Just go through the image attached above. we may understand, else leave it. Sorry for not making you understand. Thanks.

Reply:
@sureshanaparti you need to re-read the change. You'll understand there is no need of any further changes.
What @bwsw can do is say something like:
```
Make this:
# some commented stuff
to:
something not commented
```